resources/js/components/CopyButton.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div>
+    <div class="flex content-center">
         <div v-show="isNotDirty">
```

Comment:
This is great feature! Tested locally and it works great. Thank you. 

For some reason on my screen clipboard is all the way at the top of the line. 

![image](https://user-images.githubusercontent.com/436825/53463720-c4ae6b00-3a15-11e9-9c9d-376896025793.png)

Adding  `class="pt-2"` to `<div v-show="isNotDirty">` fixes that

![image](https://user-images.githubusercontent.com/436825/53463753-de4fb280-3a15-11e9-8f8c-ead9d652a10b.png)

Reply:
What browser are you using?

Reply:
Chrome Version 72.0.3626.119 (Official Build) (64-bit)

Reply:
I just pushed a fix to this branch, can you let me know if it fixed it for you? I'll merge after that.